src/main/java/uk/ac/cam/cl/dtg/segue/api/GlossaryFacade.java

```diff
@@ -43,10 +43,7 @@
 import java.util.Collections;
 import java.util.List;
 
-import static uk.ac.cam.cl.dtg.segue.api.Constants.BooleanOperator;
-import static uk.ac.cam.cl.dtg.segue.api.Constants.CONTENT_INDEX;
-import static uk.ac.cam.cl.dtg.segue.api.Constants.DEFAULT_RESULTS_LIMIT;
-import static uk.ac.cam.cl.dtg.segue.api.Constants.TYPE_FIELDNAME;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.*;
```

Comment:
!

Reply:
This is the preferred import style for the two constants classes, static and starred.